scripts/al2022/install-workarounds.sh

```diff
@@ -3,7 +3,11 @@ set -ex
 
 # update to AL2022 release that has latest repos
 # (get the latest version from dnf check-release-update)
-sudo dnf update -y --releasever=2022.0.20220204
+sudo dnf check-release-update >info 2>&1
+RELEASESERVER=$(cat info | grep "releasever*" | tail -1 | awk '{print $3}')
```

Comment:
can we delete the info file after we're done with it here?

Reply:
Yes & thanks! the file should be deleted. Will fix this.